site2/docs/client-libraries-cpp.md

```diff
@@ -104,6 +139,13 @@ If you get an error like "libpulsar.so: cannot open shared object file: No such
 
 :::
 
+### APK
+
+```bash
+apk add --allow-untrusted ./apache-pulsar-client-3.0.0-r0.apk
```

Comment:
Do we stick to 3.0.0 in the document here? If so, you may update the links above for other hard linked items (RPM, DEB).

Reply:
I think we can use `apache-pulsar-client-*.apk` instead.